controller/src/main/java/io/pravega/controller/server/eventProcessor/LocalController.java

```diff
@@ -565,6 +565,15 @@ private StreamSegments getStreamSegments(List<SegmentRange> ranges) {
                 .thenApply(status -> ModelHelper.encode(status.getState(), stream + " " + txnId));
     }
 
+    @Override
+    public AsyncIterator<io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse> listCompletedTransactions(Stream stream) {
```

Comment:
`io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse` should be imported

Reply:
Done as suggested.